File: jsimple-json/c#/jsimple/json/JsonException.cs
using System;

namespace jsimple.json
{

	using BasicException = jsimple.util.BasicException;

	/// <summary>
	/// @author Bret Johnson
	/// @since 9/15/12 5:32 PM
	/// </summary>
	public class JsonException : BasicException
	{
		public JsonException(Exception cause) : base(cause)
		{
		}

		public JsonException(string message) : base(message)
		{
		}

		public JsonException(string message, object arg1) : base(message, arg1)
		{
		}

		public JsonException(string message, object arg1, object arg2) : base(message, arg1, arg2)
		{
		}

		public JsonException(string message, params object[] args) : base(message, args)
		{
		}
	}

}

File: jsimple-json/src/main/java/jsimple/json/JsonException.java
package jsimple.json;

import jsimple.util.BasicException;

/**
 * @author Bret Johnson
 * @since 9/15/12 5:32 PM
 */
public class JsonException extends BasicException {
    public JsonException(Throwable cause) {
        super(cause);
    }

    public JsonException(String message) {
        super(message);
    }

    public JsonException(String message, Object arg1) {
        super(message, arg1);
    }

    public JsonException(String message, Object arg1, Object arg2) {
        super(message, arg1, arg2);
    }

    public JsonException(String message, Object... args) {
        super(message, args);
    }
}
